src/System.Private.CoreLib/shared/Interop/Windows/Kernel32/Interop.GetCPInfo.cs

```diff
@@ -19,6 +18,6 @@ internal unsafe struct CPINFO
         }
 
         [DllImport(Libraries.Kernel32)]
-        internal static extern unsafe int GetCPInfo(uint codePage, CPINFO* lpCpInfo);
+        internal static extern int GetCPInfo(uint codePage, out CPINFO lpCpInfo);
```

Comment:
> the return type of this should be BOOL (if you prefer interop with no marshaling overhead) or bool (if you prefer slower interop).

@jkotas, would you recommend we go through the common interop files and audit for things like this? e.g. preferring pointers over ref/out, preferring BOOL/BOOLEAN instead of int/bool, etc.?  And is there an easy-ish way of determining whether particular DllImport is resulting in the slimmest stub possible?